src/lib.rs

```diff
@@ -210,28 +212,38 @@ impl Server {
         A: ToSocketAddrs,
     {
         Server::new(ServerConfig {
-            addr,
+            addr: ConfigListenAddr::from_socket_addrs(addr)?,
             ssl: Some(config),
         })
     }
 
+    #[cfg(unix)]
+    #[inline]
```

Comment:
I know there are `#[inline]` scattered all over this project but they're really unnecessary, I should do a cleanup where I remove them...